apps/src/code-studio/pd/application_dashboard/summary_table.jsx

```diff
@@ -27,22 +38,26 @@ export default class SummaryTable extends React.Component {
     router: PropTypes.object.isRequired
   }
 
-  tableRow = (label, bgColor, textColor, data) =>  {
-    const status = label.toLowerCase().replace(/ /g,'_');
-    const total = data[status];
-
-    return (
-      <tr>
-        <td style={{backgroundColor: bgColor, color: textColor}}>{label}</td>
-        <td>{total}</td>
-      </tr>
-    );
-  };
+  tableBody = () => {
+    return Object.keys(status_colors).map((status, i) => {
+      let textColor = (status === 'declined' || status === 'unreviewed') ? color.white : color.black;
+      if (this.props.data.hasOwnProperty(status)) {
+        return (
+          <tr key={i}>
+            <td style={{backgroundColor: status_colors[status], color: textColor}}>
+              {_.upperFirst(status)}
+            </td>
+            <td>{this.props.data[status]}</td>
+          </tr>
+        );
+      }
+    });
+  }
 
   handleViewClick = (event) => {
     event.preventDefault();
     this.context.router.push(`/${this.props.path}`);
-  };
+  }
```

Comment:
The semicolon should be here, since it's an assignment (though our linting rules don't catch it, and it's not that important).